modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,137 @@
+
+Argo CD repository server (Argo CD-repo-server)::
+An Argo CD component that reads from Git repositories, generates corresponding application manifests, executes custom configuration management tooling, and returns the result to the Argo CD Application Controller. 
+
+ApplicationSet Controller (Argo CD ApplicationSet Controller)::
+A custom Kubernetes controller that exists within Argo CD and adds support for a custom `ApplicationSet` CRD.  This controller automatically generates Argo CD Applications based on the contents of an `ApplicationSet` CR and manages them, across multiple clusters declaratively, from a single mono-repository.
+
+Notifications Controller (Argo CD Notifications Controller)::
+A controller that continuously monitors Argo CD applications and provides a flexible way to notify users about important changes in the application state. 
+
+Progressive delivery::
+In the {gitops-shortname} context, progressive delivery is a process of releasing application updates in a controlled and gradual manner.
+
+Argo Rollouts::
+A controller you can use for managing the progressive deployment of applications hosted on Kubernetes and {OCP} clusters. This controller has a set of CRDs that provides advanced deployment capabilities such as blue-green, canary, canary analysis, and experimentation.
+
+Single tenancy::
+A software architecture where a single software instance serves a single user or group of users.
+
+Multitenancy::
+A software architecture where a single software instance serves multiple, distinct user groups.
+
+Local cluster::
+A cluster upon which you install Argo CD.
+
+Remote cluster::
+A cluster that you can add to Argo CD either declaratively or by using the {gitops-shortname} CLI. Remote cluster is distinct from the local cluster where Argo CD is installed on.
+
+Resource Exclusion::
+A configuration you use to exclude resources from discovery and sync so that Argo CD is unaware of them.
+
+Resource Inclusion::
+A configuration you use to include resources to discover, sync, and restrict the list of managed resources globally.
+
+ArgoCD project::
+In the {gitops-shortname} context, an Argo CD project is an entity within Argo CD that refers to the Argo CD open source project's link:https://argo-cd.readthedocs.io/en/stable/user-guide/projects[specific concept of projects], and the corresponding link:https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#projects[AppProject` CR].
++
+You may deploy Argo CD applications in an Argo CD project across different namespaces or clusters. In contrast, an link:https://docs.openshift.com/container-platform/latest/getting_started/openshift-overview.html#getting-started-openshift-common-terms_openshift-overview[OpenShift project] is necessarily restricted to a single namespace and is equivalent in concept to a namespace.
```

Comment:
```suggestion
You might deploy Argo CD applications in an Argo CD project across different namespaces or clusters. In contrast, an link:https://docs.openshift.com/container-platform/latest/getting_started/openshift-overview.html#getting-started-openshift-common-terms_openshift-overview[OpenShift project] is restricted to a single namespace and is equivalent in concept to a namespace.
```
The word "necessarily" does not add any additional meaning here, so I would omit it for minimalism purposes.